project/settings.py

```diff
@@ -194,35 +195,33 @@
     RAVEN_CONFIG = {
         'dsn': os.environ.get('SENTRY_DSN', '')
     }
+    SECURE_SSL_REDIRECT = True
```

Comment:
Esto no es correcto, se desactiva si no hay DEBUG, no si hay POSTMARK_API_KEY

Reply:
Sii, así está. Antes estaba duplicada la linea `if not DEBUG` de la linea 194

Reply:
ahh, vale, es por el diff como se ve en GH. sorryy

Reply:
jaja nada!